spec/haskell/src/SEL4/Object/FPU/X64.hs

```diff
@@ -29,10 +29,16 @@ loadFpuState tcbPtr = do
     fpuState <- asUser tcbPtr getFPUState
     doMachineOp (writeFpuState fpuState)
 
+-- We do not need all of cur_fpu_valid in the refinement proof, but do need to
+-- cross over the constraint that the current fpu owner is a tcb.
+fpuOwner_asrt :: KernelState -> Bool
+fpuOwner_asrt _ = True
+
 switchLocalFpuOwner :: Maybe (PPtr TCB) -> Kernel ()
 switchLocalFpuOwner newOwner = do
-    doMachineOp enableFpu
     curFpuOwner <- gets (x64KSCurFPUOwner . ksArchState)
+    stateAssert fpuOwner_asrt "there is a tcb at x64KSCurFPUOwner"
```

Comment:
can we please say TCB for TCBs? I know that we use tcb for code args/vars etc, but for any kind of documentation it should still be TCB